Makefile

```diff
@@ -2,7 +2,10 @@ test:
 	go test -v ./...
 
 fmt:
-	go run golang.org/x/tools/cmd/goimports@v0.3.0 -w .
+	go run golang.org/x/tools/cmd/goimports@v0.3.0 -w ./input
+	go run golang.org/x/tools/cmd/goimports@v0.3.0 -w ./model/modelprocessor
+	go run golang.org/x/tools/cmd/goimports@v0.3.0 -w ./model/internal
+	go run golang.org/x/tools/cmd/goimports@v0.3.0 -w ./model/*.go
```

Comment:
What's the reason for not formatting the generated code as well?

Reply:
`goimports` is putting stdlib import first which is conflicting with the output generated from protobuf.

To give an example with `os.pb.go`, here's a diff of the file after generation and after `goimports` was applied:

```patch
diff --git a/model/modelpb/os.pb.go b/model/modelpb/os.pb.go
index 021e22f..25f0ce6 100644
--- a/model/modelpb/os.pb.go
+++ b/model/modelpb/os.pb.go
@@ -24,10 +24,11 @@
 package modelpb
 
 import (
-       protoreflect "google.golang.org/protobuf/reflect/protoreflect"
-       protoimpl "google.golang.org/protobuf/runtime/protoimpl"
        reflect "reflect"
        sync "sync"
+
+       protoreflect "google.golang.org/protobuf/reflect/protoreflect"
+       protoimpl "google.golang.org/protobuf/runtime/protoimpl"
 )
 
 const (
```

Reply:
Why is that a problem? It should be fine to reorder to goimports style.